resources/core/values.yaml

```diff
@@ -53,11 +53,11 @@ global:
     dir: develop/
     version: 6e0a7fe2
   event_bus:
-    dir: develop/
-    version: f9df7d26
+    dir: pr/
+    version: PR-1624
   event_bus_tests:
-    dir: develop/
-    version: 1daa11b7
+    dir: pr/
+    version: PR-1624
```

Comment:
Also here, wrong PR number.

Reply:
I believe this file shouldn't be part of this PR and also the images shouldn't be merged to master where it's configured to be pulled from a PR directory.
I will remove this file of this PR, and after the merge, I will create a new PR for the tests. then a new PR for the resources.